.github/workflows/tests.yaml

```diff
@@ -14,9 +14,10 @@ jobs:
         os: [ubuntu-latest, macos-latest]
     steps:
       - uses: Homebrew/actions/setup-homebrew@master
+
       # prql-compiler
       - run: brew install prql/prql/prql-compiler
-      - run: echo "from employees | filter has_dog" | prql-compiler compile
+      - run: echo "from employees | filter has_dog" | prqlc compile
```

Comment:
Great, thanks for changing this

Reply:
FYI I adjusted this in `main` to stop it breaking.

@snth please can you review the `prql-query` part of this so we can merge this PR?

Reply:
Oh, I guess this was the approval? https://github.com/PRQL/homebrew-prql/pull/10#pullrequestreview-1262596083

Sorry, so maybe you were waiting on me!!

I'll try and merge main back in and then merge this. Sorry for the delay @pthariensflame .

Reply:
No need to worry!  I've been held up too.